bundles/org.openhab.voice.googletts/README.md

```diff
@@ -1,7 +1,7 @@
 # Google Cloud Text-to-Speech
 
-Google Cloud TTS Service uses the none-free Google Cloud Text-to-Speech API to convert text or Speech Synthesis Markup Language (SSML) input into audio data of natural human speech. 
-It provides 30 voices, available in multiple languages and variants and applies DeepMind’s groundbreaking research in WaveNet and Google’s powerful neural networks. 
+Google Cloud TTS Service uses the non-free Google Cloud Text-to-Speech API to convert text or Speech Synthesis Markup Language (SSML) input into audio data of natural human speech. 
+It provides multiple voices, available in diferent languages and variants and applies DeepMind’s groundbreaking research in WaveNet and Google’s powerful neural networks. 
```

Comment:
```suggestion
It provides multiple voices, available in different languages and variants and applies DeepMind’s groundbreaking research in WaveNet and Google’s powerful neural networks. 
```

Reply:
:roll_eyes: Good to have four eyes on it ... Thanks.